lms/static/js/Markdown.Editor.js

```diff
@@ -1061,10 +1061,45 @@
             }
         };
 
+        var hasClassList = 'classList' in document.documentElement;
+
+        // Remove class from elemnet
+        var removeClass = function (element, className) {
+            if (hasClassList) {
+                element.classList.remove(className)
+            } else {
+                element.className = (' ' + element.className + ' ')
+                    .replace(' ' + className + ' ', '')
+                    .trim()
+            }
+        };
+
+        // Add class to element
+        var addClass = function (element, className) {
+            if (hasClassList) {
+                element.classList.add(className)
+            } else {
+                var cur = ' ' + element.className + ' '
+                if (cur.indexOf(' ' + className + ' ') < 0) {
+                    element.className = (cur + className).trim()
+                }
+            }
+        };
+
+        // Checks validity of input element
+        var checkValidity = function (element) {
+            return (typeof element.checkValidity === "undefined") ? true : element.checkValidity();
+        };
+
+        // Checks the required property on element
+        var requiredPropertyCheck = function (element) {
+            return (typeof element.required === "undefined") ? $(element).attr('required') : element.required;

```

Comment:
Can we avoid JQuery usage(` $(element).attr('required')`) here since this module had no dependency on JQuery.